app/models/group.rb

```diff
@@ -0,0 +1,6 @@
+class Group < ActiveRecord::Base
+  has_many :user_groups
+  has_many :users, through: :user_groups
+
+  validates :name, presence: true, uniqueness: true
+end
```

Comment:
Z tego co zrozumiałem model ten będzie określał grupy użytkowików dot. zarówno uprawnień (admin, moderator, zwykły user), jak również roczników.

Zastanówmy się, czy nie lepiej byłoby rozdzielić ten model na dwa osobne modele. Na przykład:
1. `Role` - uprawnienia użytkowników.
2. `ExchangeSession` - model określałby rocznik aktywnej giełdy. Później, referencję do takiego modelu miałby między innymi model `Schedule`, umożliwiłoby to również definiowanie zbiorów przedmiotów powiązanych z danym rocznikiem. Ponadto przydałoby się miejsce, w którym można by zapisywać parametry giełdy, np. początkowa godzina planu zajęć - `ExchangeSession` jest na to prawidłowym kandydatem.